master/internal/trials/api_trials.go

```diff
@@ -61,15 +61,17 @@ func (a *TrialsAPIServer) StartTrial(
 			}
 		}
 
-		_, err := tx.NewUpdate().Model(&obj).WherePK().
-			Set("run_id = run_id + 1").
+		run := model.Run{ID: trialID}
+		_, err := tx.NewUpdate().Model(&run).WherePK().
```

Comment:
it'd be nice to put all these calls into a "runs" package eventually, instead of scattered over the entire codebase. can we start TODO'ing it if you agree?

Reply:
Added some TODOs